audio/engine.asm

```diff
@@ -1479,9 +1508,13 @@ Music_Call:
 	set SOUND_SUBROUTINE, [hl]
 	ret
 
-Music_Jump:
 ; jump
-; parameters: ll hh ; pointer
+; input: 
+	; bc = start of current channel
+; output: 
+	; CHANNEL_MUSIC_ADDRESS = pointer to 
+	; music address to jump to (ll hh)
```

Comment:
Again, this excess verbosity isn't helpful. Keep it all in one line. (And again, `CHANNEL_MUSIC_ADDRESS` cannot be an output. Are you sure you're understanding what's going on?)

The important part here, after all, is that the two bytes that follow the jump command itself in the channel stream represent the jump target address, in little-endian. The original comment already conveyed that information. The rest can easily be left out.